data/human/human missions.txt

```diff
@@ -3949,12 +3965,31 @@ mission "Human to Hai Space"
 			choice
 				`	(Leave.)`
 					decline
+				`	(Approach them.)`
+					to display
+						has "hai space start"
+					goto haistart
 				`	"That's your family?"`
+					to display
+						not "hai space start"
 			`	The man looks at you. "I'm adopted," he says. One of the Hai notices your confusion, and chitters. "Ah, you have not seen a human-Hai family, have you? Perhaps it is strange, but we are happy together nonetheless. Thank you for transporting our son safely."`
 			choice
 				`	"I didn't mean to be rude, I just didn't know that Hai adopted humans here."`
 				`	"No problem, I hope you enjoyed the trip."`
+			label end
 			`	After thanking you once again, they walk off. As they leave, you overhear Benjamin talk to his family about his trip. "They had so many kinds of food there, it was delicious! Although I did get homesick for pickled acorns..."`
+				decline
+			label haistart
+			`	"Of course. You did bring me all the way back here, after all. These are my parents, Yeerto and Taepip." His father Yeerto then says, "We heard about your travel agency, even went there at some point when we were thinking of going to human space. It ended up being a touch too expensive, though, so Benjamin arranged for a merchant to bring him on the way."`
+			`	"Funny we almost found each other then," he adds with a smile on his face.`
+			`	"And now look at you, a starship captain! Honest work always bears fruits," his mother Taepip tells you, before looking back at her son. "That's what we always told him."`
```

Comment:
I'm not sure this line makes sense. Why is she congratulating the player on becoming a starship captain? This is their first time meeting, supposedly.
```suggestion
```